<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en-GB">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Multiple Fill and Colour Scales in ggplot2 • ggnewscale</title>
<script src="deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="deps/headroom-0.11.0/headroom.min.js"></script><script src="deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="deps/search-1.0.0/fuse.min.js"></script><script src="deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="pkgdown.js"></script><meta property="og:title" content="Multiple Fill and Colour Scales in ggplot2">
<meta name="description" content="Use multiple fill and colour scales in ggplot2.">
<meta property="og:description" content="Use multiple fill and colour scales in ggplot2.">
<meta property="og:image" content="https://eliocamp.github.io/ggnewscale/logo.png">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="index.html">ggnewscale</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.5.2.9000</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="reference/index.html">Reference</a></li>
<li class="nav-item"><a class="nav-link" href="news/index.html">Changelog</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="search.json">
</form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/eliocamp/ggnewscale/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-home">
<div class="row">
  <main id="main" class="col-md-9"><div class="section level1">
<div class="page-header">
<img src="logo.png" class="logo" alt=""><h1 id="ggnewscale-">ggnewscale <a class="anchor" aria-label="anchor" href="#ggnewscale-"></a>
</h1>
</div>
<!-- badges: start -->

<p><code>ggnewscale</code> tries to make it painless to use multiple scales in <code>ggplot2</code>. Although originally intended to use with colour and fill, it should work with any <code>aes</code>, such as <code>shape</code>, <code>linetype</code> and the rest.</p>
<p><a href="https://web.archive.org/web/20220511154749/https://twitter.com/mattansb/status/1524415881920528385" class="external-link">ggnewscale: spend 400% more time tweaking your ggplot!</a></p>
<p>For another way of defining multiple scales, you can also try <a href="https://github.com/clauswilke/relayer" class="external-link">relayer</a>.</p>
<div class="section level2">
<h2 id="how-to-install">How to install<a class="anchor" aria-label="anchor" href="#how-to-install"></a>
</h2>
<p>You can install ggnewscale from CRAN with:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"ggnewscale"</span><span class="op">)</span></span></code></pre></div>
<p>Or the development version with:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># install.packages("devtools")</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">"eliocamp/ggnewscale"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="how-to-cite">How to cite<a class="anchor" aria-label="anchor" href="#how-to-cite"></a>
</h2>
<p>If you use ggnewscale in a publication, I’ll be grateful if you cited it. To get the suggested citation for this (and any other R package) you can use:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/citation.html" class="external-link">citation</a></span><span class="op">(</span><span class="st">"ggnewscale"</span><span class="op">)</span></span>
<span><span class="co">#&gt; To cite ggnewscale in publications use:</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   Campitelli E (????). _ggnewscale: Multiple Fill and Colour Scales in</span></span>
<span><span class="co">#&gt;   'ggplot2'_. doi:10.5281/zenodo.2543762</span></span>
<span><span class="co">#&gt;   &lt;https://doi.org/10.5281/zenodo.2543762&gt;, R package version 0.5.2.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; A BibTeX entry for LaTeX users is</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   @Manual{R-ggnewscale,</span></span>
<span><span class="co">#&gt;     title = {ggnewscale: Multiple Fill and Colour Scales in 'ggplot2'},</span></span>
<span><span class="co">#&gt;     author = {Elio Campitelli},</span></span>
<span><span class="co">#&gt;     note = {R package version 0.5.2},</span></span>
<span><span class="co">#&gt;     doi = {10.5281/zenodo.2543762},</span></span>
<span><span class="co">#&gt;   }</span></span></code></pre></div>
<p>If you use knitr, <a href="https://bookdown.org/yihui/rmarkdown-cookbook/write-bib.html" class="external-link">you can automate this</a> with</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/write_bib.html" class="external-link">write_bib</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"ggnewscale"</span><span class="op">)</span>, <span class="st">"packages.bib"</span><span class="op">)</span></span></code></pre></div>
<p>And then add citations with <code>@R-ggnewscale</code>.</p>
<details><summary>
Click to see a list of some publications that have cited ggnewscale. Thanks!
</summary><p>[1] <em>RET Kinase Is Transcriptionally Regulated by ASCL1 in Neuroendocrine Prostate Cancer and Small Cell Lung Cancer - ProQuest</em>. <a href="https://www.proquest.com/openview/1c55b283c4260b1202b8b755862c78fc/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y" class="external-link uri">https://www.proquest.com/openview/1c55b283c4260b1202b8b755862c78fc/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y</a>. Sep. 04, 2024. (Visited on 09/04/2024).</p>
<p>[2] <em>Application of Rapid Diagnostics and Next-Generation Amplicon Sequencing to Address Product Quality in the Food Industry - ProQuest</em>. <a href="https://www.proquest.com/openview/3238c502f2c2d6c8bef42493c35d6a47/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y" class="external-link uri">https://www.proquest.com/openview/3238c502f2c2d6c8bef42493c35d6a47/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y</a>. Jun. 28, 2023. (Visited on 06/28/2023).</p>
<p>[3] <em>Exploring Development and Genetic Variation Within <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&lt;</mo><annotation encoding="application/x-tex">&lt;</annotation></semantics></math>em<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&gt;</mo><annotation encoding="application/x-tex">&gt;</annotation></semantics></math>Vitis<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&lt;</mo><annotation encoding="application/x-tex">&lt;</annotation></semantics></math>/Em<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&gt;</mo><annotation encoding="application/x-tex">&gt;</annotation></semantics></math> - ProQuest</em>. <a href="https://www.proquest.com/openview/34d303f27deac557b3314eb9887b8ca9/1.pdf?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y%5C#page=34" class="external-link uri">https://www.proquest.com/openview/34d303f27deac557b3314eb9887b8ca9/1.pdf?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y#page=34</a>. Aug. 04, 2024. (Visited on 08/04/2024).</p>
<p>[4] <em>Integrated Analysis of Hub Genes and miRNA- Transcription Factor-Hub Gene Interaction Network in Necrotizing Enterocolitis</em>. <a href="https://www.researchsquare.com" class="external-link uri">https://www.researchsquare.com</a>. Apr. 2023. DOI: 10.21203/rs.3.rs-2756663/v1. (Visited on 04/12/2023).</p>
<p>[5] <em>Leveraging Genome Evolution to Address Questions of Biological Function - ProQuest</em>. <a href="https://www.proquest.com/openview/db93724e3d81708ab414e6afaa8271b4/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y" class="external-link uri">https://www.proquest.com/openview/db93724e3d81708ab414e6afaa8271b4/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y</a>. Aug. 04, 2024. (Visited on 08/04/2024).</p>
<p>[6] <em>Nutrigonometry II: Experimental Strategies to Maximize Nutritional Information in Multidimensional Performance Landscapes - Morimoto - 2022 - Ecology and Evolution - Wiley Online Library</em>. <a href="https://onlinelibrary.wiley.com/doi/10.1002/ece3.9174" class="external-link uri">https://onlinelibrary.wiley.com/doi/10.1002/ece3.9174</a>. Aug. 08, 2022. (Visited on 08/08/2022).</p>
<p>[7] <em>Rifaximin Prophylaxis Causes Resistance to the Last-Resort Antibiotic Daptomycin medRxiv</em>. <a href="https://www.medrxiv.org/content/10.1101/2023.03.01.23286614v1" class="external-link uri">https://www.medrxiv.org/content/10.1101/2023.03.01.23286614v1</a>. Mar. 10, 2023. (Visited on 03/10/2023).</p>
<p>[8] <em>Study on the Mechanism of Lung Ischemia-Reperfusion Injury Based on Bioinformatics</em>. <a href="https://www.researchsquare.com" class="external-link uri">https://www.researchsquare.com</a>. Jul. 2023. DOI: 10.21203/rs.3.rs-3088383/v1. (Visited on 07/19/2023).</p>
<p>[9] <em>Using Stochastic Point Pattern Analysis to Track Regional Orientations of Magmatism During the Transition to Cenozoic Extension and Rio Grande Rifting, Southern Rocky Mountains - Rosera - 2024 - Tectonics - Wiley Online Library</em>. <a href="https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2023TC007902" class="external-link uri">https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2023TC007902</a>. Feb. 12, 2024. (Visited on 02/12/2024).</p>
<p>[10] <em>Visual Integration of Genome-Wide Association Studies and Differential Expression Results with the Hidecan R Package</em>. <a href="https://www.mdpi.com/2073-4425/15/10/1244" class="external-link uri">https://www.mdpi.com/2073-4425/15/10/1244</a>. Oct. 01, 2024. (Visited on 10/01/2024).</p>
<p>[11] E. Akhil Prakash, T. Hromádková, T. Jabir, et al. “Dissemination of Multidrug Resistant Bacteria to the Polar Environment - Role of the Longest Migratory Bird Arctic Tern (Sterna Paradisaea)”. In: <em>Science of The Total Environment</em> (Dec. 2021), p. 152727. ISSN: 0048-9697. DOI: 10.1016/j.scitotenv.2021.152727. (Visited on 01/03/2022).</p>
<p>[12] R. AminiTabrizi, R. M. Wilson, J. D. Fudyma, et al. “Controls on Soil Organic Matter Degradation and Subsequent Greenhouse Gas Emissions Across a Permafrost Thaw Gradient in Northern Sweden”. In: <em>Frontiers in Earth Science</em> 8 (2020). ISSN: 2296-6463. DOI: 10.3389/feart.2020.557961. (Visited on 03/03/2021).</p>
<p>[13] A. André, J. Plantade, I. Durieux, et al. “Genomics Unveils Country-to-Country Transmission between Animal Hospitals of a Multidrug-Resistant and Sequence Type 2 Acinetobacter Baumannii Clone”. In: <em>Microbial Genomics</em> 10.10 (2024), p. 001292. ISSN: 2057-5858. DOI: 10.1099/mgen.0.001292. (Visited on 10/20/2024).</p>
<p>[14] D. Baker, J. Lauer, A. Ortega, et al. “Effects of Phycosphere Bacteria on Their Algal Host Are Host Species-Specific and Not Phylogenetically Conserved”. In: <em>Microorganisms</em> 11.1 (Jan. 2023), p. 62. ISSN: 2076-2607. DOI: 10.3390/microorganisms11010062. (Visited on 12/28/2022).</p>
<p>[15] L. Baumgarten, B. Pieper, B. Song, et al. “Pan-European Study of Genotypes and Phenotypes in the Arabidopsis Relative Cardamine Hirsuta Reveals How Adaptation, Demography, and Development Shape Diversity Patterns”. In: <em>PLOS Biology</em> 21.7 (Jul. 2023), p. e3002191. ISSN: 1545-7885. DOI: 10.1371/journal.pbio.3002191. (Visited on 07/24/2023).</p>
<p>[16] J. R. M. Black, G. Bartha, C. W. Abbott, et al. “Ultrasensitive ctDNA Detection for Preoperative Disease Stratification in Early-Stage Lung Adenocarcinoma”. In: <em>Nature Medicine</em> (Jan. 2025), pp. 1-7. ISSN: 1546-170X. DOI: 10.1038/s41591-024-03216-y. (Visited on 01/16/2025).</p>
<p>[17] J. Bosch, P. T. Dobbler, T. Větrovský, et al. “Decomposition of Fomes Fomentatius Fruiting Bodies - Transition of Healthy Living Fungus into a Decayed Bacteria-Rich Habitat Is Primarily Driven by Arthropoda”. In: <em>FEMS Microbiology Ecology</em> (Mar. 2024), p. fiae044. ISSN: 0168-6496. DOI: 10.1093/femsec/fiae044. (Visited on 04/05/2024).</p>
<p>[18] J. Botero, C. Peeters, E. De Canck, et al. “A Comparative Genomic Analysis of _Fructobacillus__ Evanidus_ Sp. Nov. from Bumble Bees”. In: <em>Systematic and Applied Microbiology</em> (Mar. 2024), p. 126505. ISSN: 0723-2020. DOI: 10.1016/j.syapm.2024.126505. (Visited on 03/29/2024).</p>
<p>[19] J. Botero, A. S. Sombolestani, M. Cnockaert, et al. “A Phylogenomic and Comparative Genomic Analysis of Commensalibacter, a Versatile Insect Symbiont”. In: <em>Animal Microbiome</em> 5.1 (Apr. 2023), p. 25. ISSN: 2524-4671. DOI: 10.1186/s42523-023-00248-6. (Visited on 05/02/2023).</p>
<p>[20] I. N. Boys, A. G. Johnson, M. R. Quinlan, et al. “Structural Homology Screens Reveal Host-Derived Poxvirus Protein Families Impacting Inflammasome Activity”. In: <em>Cell Reports</em> 42.8 (Aug. 2023), p. 112878. ISSN: 2211-1247. DOI: 10.1016/j.celrep.2023.112878. (Visited on 07/30/2023).</p>
<p>[21] I. N. Boys, A. G. Johnson, M. Quinlan, et al. <em>Structural Homology Screens Reveal Poxvirus-Encoded Proteins Impacting Inflammasome-Mediated Defenses</em>. Feb. 2023. DOI: 10.1101/2023.02.26.529821. (Visited on 03/02/2023).</p>
<p>[22] S. Bruns, S. Al Hameli, E. Sulanke, et al. “A Wandering Wedgefish Illustrates the Need for Cooperative Elasmobranch Conservation in the Arabian Gulf”. In: <em>Environmental Biology of Fishes</em> (Mar. 2024). ISSN: 1573-5133. DOI: 10.1007/s10641-024-01531-4. (Visited on 03/09/2024).</p>
<p>[23] N. H. Buitendijk and B. A. Nolet. “Timing and Intensity of Goose Grazing: Implications for Grass Height and First Harvest”. In: <em>Agriculture, Ecosystems &amp; Environment</em> 357 (Nov. 2023), p. 108681. ISSN: 0167-8809. DOI: 10.1016/j.agee.2023.108681. (Visited on 07/25/2023).</p>
<p>[24] F. Cai, S. Mao, S. Peng, et al. “A Comprehensive Pan-Cancer Examination of Transcription Factor MAFF: Oncogenic Potential, Prognostic Relevance, and Immune Landscape Dynamics”. In: <em>International Immunopharmacology</em> 149 (Mar. 2025), p. 114105. ISSN: 1878-1705. DOI: 10.1016/j.intimp.2025.114105.</p>
<p>[25] L. F. Camacho, J. E. Ávila, and C. Flórez-V. “Mechanisms Preventing Animals to Achieve Buoyant Flight”. In: <em>Journal of Natural History</em> 58.9-12 (Mar. 2024), pp. 440-448. ISSN: 0022-2933. DOI: 10.1080/00222933.2024.2322178. (Visited on 03/29/2024).</p>
<p>[26] Y. Cao, Z. Shen, N. Zhang, et al. “Phosphorus Availability Influences Disease-Suppressive Soil Microbiome through Plant-Microbe Interactions”. In: <em>Microbiome</em> 12.1 (Sep. 2024), p. 185. ISSN: 2049-2618. DOI: 10.1186/s40168-024-01906-w. (Visited on 10/01/2024).</p>
<p>[27] M. E. Carey, Z. A. Dyson, D. J. Ingle, et al. “Global Diversity and Antimicrobial Resistance of Typhoid Fever Pathogens: Insights from a Meta-Analysis of 13,000 Salmonella Typhi Genomes”. In: <em>eLife</em> 12 (Sep. 2023), p. e85867. ISSN: 2050-084X. DOI: 10.7554/eLife.85867. (Visited on 09/19/2023).</p>
<p>[28] H. Chen, G. Chew, N. Devapragash, et al. “The E3 Ubiquitin Ligase WWP2 Regulates Pro-Fibrogenic Monocyte Infiltration and Activity in Heart Fibrosis”. In: <em>Nature Communications</em> 13.1 (Nov. 2022), p. 7375. ISSN: 2041-1723. DOI: 10.1038/s41467-022-34971-6. (Visited on 12/03/2022).</p>
<p>[29] T. Chen, Y. Lei, M. Li, et al. “Network Pharmacology to Unveil the Mechanism of Suanzaoren Decoction in the Treatment of Alzheimer’s with Diabetes”. In: <em>Hereditas</em> 161.1 (Jan. 2024), p. 2. ISSN: 1601-5223. DOI: 10.1186/s41065-023-00301-z.</p>
<p>[30] H. Danlin, T. Chao, S. Zhenjie, et al. “Potential Synergistic Regulation of Hsp70 and Antioxidant Enzyme Genes in Pyropia Yezoensis under High Temperature Stress”. In: <em>Algal Research</em> (Jan. 2024), p. 103375. ISSN: 2211-9264. DOI: 10.1016/j.algal.2023.103375. (Visited on 01/16/2024).</p>
<p>[31] K. Darragh, K. M. Kay, and S. R. Ramírez. “The Convergent Evolution of Hummingbird Pollination Results in Repeated Floral Scent Loss through Gene Downregulation”. In: <em>Molecular Biology and Evolution</em> (Feb. 2025), p. msaf027. ISSN: 1537-1719. DOI: 10.1093/molbev/msaf027.</p>
<p>[32] X. Ding, K. Liu, Q. Yan, et al. “Sugar and Organic Acid Availability Modulate Soil Diazotroph Community Assembly and Species Co-Occurrence Patterns on the Tibetan Plateau”. In: <em>Applied Microbiology and Biotechnology</em> (Oct. 2021). ISSN: 1432-0614. DOI: 10.1007/s00253-021-11629-9. (Visited on 10/21/2021).</p>
<p>[33] T. G. Drivas, A. Lucas, and M. D. Ritchie. “eQTpLot: A User-Friendly R Package for the Visualization of Colocalization between eQTL and GWAS Signals”. In: <em>BioData Mining</em> 14.1 (Jul. 2021), p. 32. ISSN: 1756-0381. DOI: 10.1186/s13040-021-00267-6. (Visited on 07/21/2021).</p>
<p>[34] Z. Feng, P. Hao, Y. Yang, et al. “Network Pharmacology and Molecular Docking to Explore the Potential Molecular Mechanism of Chlorogenic Acid Treatment of Oral Squamous Cell Carcinoma”. In: <em>Medicine</em> 103.45 (Nov. 2024), p. e40218. DOI: 10.1097/MD.0000000000040218. (Visited on 11/15/2024).</p>
<p>[35] M. Gao, T. Wang, X. Lin, et al. “Retinal Organoid Microenvironment Enhanced Bioactivities of Microglia-Like Cells Derived From HiPSCs”. In: <em>Investigative Ophthalmology &amp; Visual Science</em> 65.12 (Oct. 2024), p. 19. ISSN: 1552-5783. DOI: 10.1167/iovs.65.12.19. (Visited on 10/14/2024).</p>
<p>[36] K. Giannakis, S. J. Arrowsmith, L. Richards, et al. “Evolutionary Inference across Eukaryotes Identifies Universal Features Shaping Organelle Gene Retention”. In: <em>Cell Systems</em> (Sep. 2022). ISSN: 2405-4712. DOI: 10.1016/j.cels.2022.08.007. (Visited on 09/19/2022).</p>
<p>[37] R. B. Gorodnichev, M. A. Kornienko, M. V. Malakhova, et al. “Isolation and Characterization of the First Zobellviridae Family Bacteriophage Infecting Klebsiella Pneumoniae”. In: <em>International Journal of Molecular Sciences</em> 24.4 (Jan. 2023), p. 4038. ISSN: 1422-0067. DOI: 10.3390/ijms24044038. (Visited on 02/20/2023).</p>
<p>[38] M. C. Granovetter, L. Ettensohn, and M. Behrmann. “With Childhood Hemispherectomy, One Hemisphere Can Support-But Is Suboptimal for-Word and Face Recognition”. In: <em>bioRxiv</em> (Nov. 2020), p. 2020.11.06.371823. DOI: 10.1101/2020.11.06.371823. (Visited on 03/03/2021).</p>
<p>[39] T. Grubinger. “Inference of Population History and Spatio-Temporal Patterns of Diversity in Wild and Cultivated Tomatoes”. PhD thesis. University of Zurich, Feb. 2025. DOI: 10.5167/UZH-270864. (Visited on 02/11/2025).</p>
<p>[40] M. Heuberger, D. Koo, H. I. Ahmed, et al. “Evolution of Einkorn Wheat Centromeres Is Driven by the Mutualistic Interplay of Two LTR Retrotransposons”. In: <em>Mobile DNA</em> 15.1 (Aug. 2024), p. 16. ISSN: 1759-8753. DOI: 10.1186/s13100-024-00326-9. (Visited on 08/08/2024).</p>
<p>[41] C. Higgs, L. S. Kumar, K. Stevens, et al. “Comparison of Contemporary Invasive and Non-Invasive Streptococcus Pneumoniae Isolates Reveals New Insights into Circulating Anti-Microbial Resistance Determinants”. In: <em>Antimicrobial Agents and Chemotherapy</em> 0.0 (Oct. 2023), pp. e00785-23. DOI: 10.1128/aac.00785-23. (Visited on 10/23/2023).</p>
<p>[42] C. Higgs, L. S. Kumar, K. Stevens, et al. “Population Structure, Serotype Distribution and Antibiotic Resistance of Streptococcus Pneumoniae Causing Invasive Disease in Victoria, Australia”. In: <em>Microbial Genomics</em> 9.7 (2023), p. 001070. ISSN: 2057-5858. DOI: 10.1099/mgen.0.001070. (Visited on 07/25/2023).</p>
<p>[43] R. Hill, M. Grey, M. O. Fedi, et al. <em>Evolutionary Genomics Reveals Variation in Structure and Genetic Content Implicated in Virulence and Lifestyle in the Genus Gaeumannomyces</em>. Feb. 2024. DOI: 10.1101/2024.02.15.580261. (Visited on 02/20/2024).</p>
<p>[44] A. T. Hinsu, K. J. Panchal, R. J. Pandit, et al. “Characterizing Rhizosphere Microbiota of Peanut (Arachis Hypogaea L.) from Pre-Sowing to Post-Harvest of Crop under Field Conditions”. In: <em>Scientific Reports</em> 11.1 (Aug. 2021), p. 17457. ISSN: 2045-2322. DOI: 10.1038/s41598-021-97071-3. (Visited on 09/06/2021).</p>
<p>[45] T. Hinzke, F. Tanneberger, C. Aggenbach, et al. “Response Patterns of Fen Sedges to a Nutrient Gradient Indicate Both Geographic Origin-Specific Genotypic Differences and Phenotypic Plasticity”. In: <em>Wetlands</em> 42.8 (Nov. 2022), p. 113. ISSN: 1943-6246. DOI: 10.1007/s13157-022-01629-4. (Visited on 11/24/2022).</p>
<p>[46] E. Järvi-Laturi, T. Tahvanainen, E. Koskinen, et al. “Plant Community Composition Controls Spatial Variation in Year-Round Methane Fluxes in a Boreal Rich Fen”. In: <em>EGUsphere</em> 2025 (2025), pp. 1-36. DOI: 10.5194/egusphere-2025-217.</p>
<p>[47] M. Jenckel, I. Smith, T. King, et al. “Distribution and Genetic Diversity of Hepatitis E Virus in Wild and Domestic Rabbits in Australia”. In: <em>Pathogens</em> 10.12 (Dec. 2021), p. 1637. DOI: 10.3390/pathogens10121637. (Visited on 12/21/2021).</p>
<p>[48] H. Jentsch and J. Weidinger. “Spatio-Temporal Analysis of Valley Wind Systems in the Complex Mountain Topography of the Rolwaling Himal, Nepal”. In: <em>Atmosphere</em> 13.7 (Jul. 2022), p. 1138. ISSN: 2073-4433. DOI: 10.3390/atmos13071138. (Visited on 08/01/2022).</p>
<p>[49] B. Jiang, D. M. Weinstock, K. A. Donovan, et al. “ITK Degradation to Block T Cell Receptor Signaling and Overcome Therapeutic Resistance in T Cell Lymphomas”. In: <em>Cell Chemical Biology</em> 0.0 (Apr. 2023). ISSN: 2451-9456, 2451-9448. DOI: 10.1016/j.chembiol.2023.03.007. (Visited on 04/07/2023).</p>
<p>[50] X. Jin, F. B. Yu, J. Yan, et al. “Culturing of a Complex Gut Microbial Community in Mucin-Hydrogel Carriers Reveals Strain- and Gene-Associated Spatial Organization”. In: <em>Nature Communications</em> 14.1 (Jun. 2023), p. 3510. ISSN: 2041-1723. DOI: 10.1038/s41467-023-39121-0. (Visited on 06/17/2023).</p>
<p>[51] P. M. Joubert. “Catching up to Fungal Plant Pathogens: A Characterization of Extrachromosomal Circular DNAs and Gene Presence Absence Variation in Magnaporthe Oryzae”. PhD thesis. UC Berkeley, 2023. (Visited on 09/28/2023).</p>
<p>[52] P. M. Joubert and K. V. Krasileva. <em>Distinct Genomic Contexts Predict Gene Presence-Absence Variation in Different Pathotypes of a Fungal Plant Pathogen</em>. Feb. 2023. DOI: 10.1101/2023.02.17.529015. (Visited on 02/20/2023).</p>
<p>[53] M. Jung, D. Wells, J. Rusch, et al. “Unified Single-Cell Analysis of Testis Gene Regulation and Pathology in Five Mouse Strains”. In: <em>eLife</em> 8 (Jun. 2019). Ed. by D. Bourc’his, P. J. Wittkopp and S. Lukassen, p. e43966. ISSN: 2050-084X. DOI: 10.7554/eLife.43966. (Visited on 03/03/2021).</p>
<p>[54] T. Karasaki, D. A. Moore, S. Veeriah, et al. “Evolutionary Characterization of Lung Adenocarcinoma Morphology in TRACERx”. In: <em>Nature Medicine</em> (Apr. 2023), pp. 1-13. ISSN: 1546-170X. DOI: 10.1038/s41591-023-02230-w. (Visited on 04/14/2023).</p>
<p>[55] N. Khan, H. T. T. Nguyen, S. Galelli, et al. “Increasing Drought Risks Over the Past Four Centuries Amidst Projected Flood Intensification in the Kabul River Basin (Afghanistan and Pakistan)-Evidence From Tree Rings”. In: <em>Geophysical Research Letters</em> 49.24 (2022), p. e2022GL100703. ISSN: 1944-8007. DOI: 10.1029/2022GL100703. (Visited on 05/09/2023).</p>
<p>[56] M. Kornienko, D. Bespiatykh, M. Malakhova, et al. “PCR Assay for Rapid Taxonomic Differentiation of Virulent Staphylococcus Aureus and Klebsiella Pneumoniae Bacteriophages”. In: <em>International Journal of Molecular Sciences</em> 24.5 (Jan. 2023), p. 4483. ISSN: 1422-0067. DOI: 10.3390/ijms24054483. (Visited on 03/02/2023).</p>
<p>[57] S. Kudrenko, J. Vollering, A. Zedrosser, et al. “Walking on the Dark Side: Anthropogenic Factors Limit Suitable Habitat for Gray Wolf (Canis Lupus) in a Large Natural Area Covering Belarus and Ukraine”. In: <em>Global Ecology and Conservation</em> (Jul. 2023), p. e02586. ISSN: 2351-9894. DOI: 10.1016/j.gecco.2023.e02586. (Visited on 07/27/2023).</p>
<p>[58] R. A. Laing, M. J. Foster, M. A. Hassani, et al. <em>Complex Exchanges among Plasmids and Clonal Expansion of Lineages Shape the Population Structure and Virulence of Borrelia Burgdorferi</em>. Jan. 2025. DOI: 10.1101/2025.01.29.635312. (Visited on 02/05/2025).</p>
<p>[59] A. Lan, K. Kang, S. Tang, et al. “Fine-Scale Population Structure and Demographic History of Han Chinese Inferred from Haplotype Network of 111,000 Genomes”. In: <em>bioRxiv</em> (Jul. 2020), p. 2020.07.03.166413. DOI: 10.1101/2020.07.03.166413. (Visited on 03/03/2021).</p>
<p>[60] Z. Lapp, R. Crawford, A. Miles-Jay, et al. “Regional Spread of blaNDM-1-containing Klebsiella Pneumoniae ST147 in Post-Acute Care Facilities”. In: <em>Clinical Infectious Diseases</em> (May. 2021). ISSN: 1058-4838. DOI: 10.1093/cid/ciab457. (Visited on 05/21/2021).</p>
<p>[61] J. Lechmann, A. Szelecsenyi, S. Bruhn, et al. <em>The Swiss National Program for the Surveillance of Influenza A Viruses in Pigs and Humans: Genetic Variability and Zoonotic Transmissions from 2010 - 2022</em>. Jan. 2025. DOI: 10.1101/2025.01.28.24319114. (Visited on 02/05/2025).</p>
<p>[62] J. Lechmann, A. Szelecsenyi, S. Bruhn, et al. <em>The Swiss National Program for the Surveillance of Influenza A Viruses in Pigs and Humans: Genetic Variability and Zoonotic Transmissions from 2010 - 2022</em>. Jan. 2025. DOI: 10.1101/2025.01.28.24319114. (Visited on 01/31/2025).</p>
<p>[63] L. S. H. Lee and C. Y. Jim. “Thermal and Humidification Effects of a Swimming Pool in Hong Kong’s Humid-Subtropical Summer”. In: <em>Sustainable Cities and Society</em> (Jul. 2023), p. 104816. ISSN: 2210-6707. DOI: 10.1016/j.scs.2023.104816. (Visited on 07/25/2023).</p>
<p>[64] Z. Liang, T. Liu, Q. Li, et al. “Deciphering the Functional Landscape of Phosphosites with Deep Neural Network”. In: <em>Cell Reports</em> 42.9 (Sep. 2023). ISSN: 2211-1247. DOI: 10.1016/j.celrep.2023.113048. (Visited on 09/19/2023).</p>
<p>[65] X. Lin, Z. Sha, J. Trimpert, et al. “The NSP4 T492I Mutation Increases SARS-CoV-2 Infectivity by Altering Non-Structural Protein Cleavage”. In: <em>Cell Host &amp; Microbe</em> (Jul. 2023). ISSN: 1931-3128. DOI: 10.1016/j.chom.2023.06.002. (Visited on 07/06/2023).</p>
<p>[66] J. Ma, X. Zhu, R. Hu, et al. “A Systematic Review, Meta-Analysis and Meta-Regression of the Global Prevalence of Foodborne Vibrio Spp. Infection in Fishes: A Persistent Public Health Concern”. In: <em>Marine Pollution Bulletin</em> 187 (Feb. 2023), p. 114521. ISSN: 0025-326X. DOI: 10.1016/j.marpolbul.2022.114521. (Visited on 01/10/2023).</p>
<p>[67] X. Ma, H. Ren, R. Peng, et al. “Identification of Key Genes Associated with Progression and Prognosis for Lung Squamous Cell Carcinoma”. In: <em>PeerJ</em> 8 (May. 2020), p. e9086. ISSN: 2167-8359. DOI: 10.7717/peerj.9086. (Visited on 02/12/2024).</p>
<p>[68] D. G. Maghini, M. Dvorak, A. Dahlen, et al. “Quantifying Bias Introduced by Sample Collection in Relative and Absolute Microbiome Measurements”. In: <em>Nature Biotechnology</em> (Apr. 2023), pp. 1-11. ISSN: 1546-1696. DOI: 10.1038/s41587-023-01754-3. (Visited on 05/02/2023).</p>
<p>[69] A. Maqueda-Real, L. Ollé-Monràs, and S. Park. <em>Mapping Cancer Gene Dynamics through State-Specific Interactions</em>. Jan. 2025. DOI: 10.1101/2025.01.08.631858. (Visited on 01/16/2025).</p>
<p>[70] S. Mehic. “Genomic and Physiologic Characterization of a Freshwater Photoarsenotroph, Cereibacter Azotoformans Str. ORIO, Isolated From Sediments Capable of Light-Dark Arsenic Redox Cycling”. PhD thesis. UC Santa Cruz, 2022. (Visited on 05/09/2023).</p>
<p>[71] E. Merino Tejero, D. Lashgari, R. García-Valiente, et al. “Multiscale Modeling of Germinal Center Recapitulates the Temporal Transition From Memory B Cells to Plasma Cells Differentiation as Regulated by Antigen Affinity-Based Tfh Cell Help”. In: <em>Frontiers in Immunology</em> 11 (Feb. 2021). ISSN: 1664-3224. DOI: 10.3389/fimmu.2020.620716. (Visited on 03/03/2021).</p>
<p>[72] I. Mokrousov, A. Vyazovaya, E. Shitikov, et al. “Insight into Pathogenomics and Phylogeography of Hypervirulent and Highly-Lethal Mycobacterium Tuberculosis Strain Cluster”. In: <em>BMC Infectious Diseases</em> 23.1 (Jun. 2023), p. 426. ISSN: 1471-2334. DOI: 10.1186/s12879-023-08413-7. (Visited on 06/28/2023).</p>
<p>[73] R. D. Moore. “Fitting Power-Law Relations in Watershed Science and Analysis, with an Example Using the R Language”. In: <em>Confluence: Journal of Watershed Science and Management</em> 7.1 (Feb. 2024), pp. 14 pp-14 pp. ISSN: 2561-1836. DOI: 10.22230/jwsm.2024v7n1a53. (Visited on 02/20/2024).</p>
<p>[74] I. Navickaite, H. Holmes, L. Dondi, et al. “Occurrence and Characterization of rmtB-harbouring Salmonella and Escherichia Coli Isolates from a Pig Farm in the UK”. In: <em>Journal of Antimicrobial Chemotherapy</em> (Apr. 2024), p. dkae102. ISSN: 0305-7453. DOI: 10.1093/jac/dkae102. (Visited on 04/19/2024).</p>
<p>[75] G. Papacharalampous, H. Tyralis, S. M. Papalexiou, et al. “Global-Scale Massive Feature Extraction from Monthly Hydroclimatic Time Series: Statistical Characterizations, Spatial Patterns and Hydrological Similarity”. In: <em>Science of The Total Environment</em> 767 (May. 2021), p. 144612. ISSN: 0048-9697. DOI: 10.1016/j.scitotenv.2020.144612. (Visited on 03/03/2021).</p>
<p>[76] Y. Peng, Z. Lu, D. Pan, et al. “Viruses in Deep-Sea Cold Seep Sediments Harbor Diverse Survival Mechanisms and Remain Genetically Conserved within Species”. In: <em>The ISME Journal</em> 17.10 (Oct. 2023), pp. 1774-1784. ISSN: 1751-7370. DOI: 10.1038/s41396-023-01491-0. (Visited on 04/11/2024).</p>
<p>[77] C. Plaza, P. García-Palacios, A. A. Berhe, et al. “Ecosystem Productivity Has a Stronger Influence than Soil Age on Surface Soil Carbon Storage across Global Biomes”. In: <em>Communications Earth &amp; Environment</em> 3.1 (Oct. 2022), pp. 1-8. ISSN: 2662-4435. DOI: 10.1038/s43247-022-00567-7. (Visited on 10/12/2022).</p>
<p>[78] P. Pottier, H. Lin, R. R. Y. Oh, et al. “A Comprehensive Database of Amphibian Heat Tolerance”. In: <em>Scientific Data</em> 9.1 (Oct. 2022), p. 600. ISSN: 2052-4463. DOI: 10.1038/s41597-022-01704-9. (Visited on 10/08/2022).</p>
<p>[79] J. M. Quilty, A. E. Sikorska-Senoner, and D. Hah. “A Stochastic Conceptual-Data-Driven Approach for Improved Hydrological Simulations”. In: <em>Environmental Modelling &amp; Software</em> (Jan. 2022), p. 105326. ISSN: 1364-8152. DOI: 10.1016/j.envsoft.2022.105326. (Visited on 01/19/2022).</p>
<p>[80] D. L. Rios, P. C. L. da Silva, C. S. S. Moura, et al. “Comparative Metatranscriptome Analysis of Brazilian Milk and Water Kefir Beverages”. In: <em>International Microbiology</em> (Sep. 2023). ISSN: 1618-1905. DOI: 10.1007/s10123-023-00431-4. (Visited on 10/03/2023).</p>
<p>[81] H. Rodenhizer, F. Belshe, G. Celis, et al. “Abrupt Permafrost Thaw Accelerates Carbon Dioxide and Methane Release at a Tussock Tundra Site”. In: <em>Arctic, Antarctic, and Alpine Research</em> 54.1 (Dec. 2022), pp. 443-464. ISSN: 1523-0430. DOI: 10.1080/15230430.2022.2118639. (Visited on 10/04/2022).</p>
<p>[82] E. S. Røed. “Evaluating Genetic Tools to Inform Conservation Efforts for the Nationally Red-Listed European Lobster (Homarus Gammarus)”. MA Thesis. Norwegian University of Life Sciences, 2024. (Visited on 08/29/2024).</p>
<p>[83] A. Rutz, M. Sorokina, J. Galgonek, et al. “Open Natural Products Research: Curation and Dissemination of Biological Occurrences of Chemical Structures through Wikidata”. In: <em>bioRxiv</em> (Mar. 2021), p. 2021.02.28.433265. DOI: 10.1101/2021.02.28.433265. (Visited on 03/07/2021).</p>
<p>[84] A. Saha, A. Bellucci, S. Fratini, et al. “Ecological Factors and Parity Mode Correlate with Genome Size Variation in Squamate Reptiles”. In: <em>BMC Ecology and Evolution</em> 23.1 (Dec. 2023), p. 69. ISSN: 2730-7182. DOI: 10.1186/s12862-023-02180-4. (Visited on 12/16/2023).</p>
<p>[85] M. R. Scharn, M. C. G. Brachmann, M. A. Patchett, et al. <em>Vegetation Responses to 26 Years of Warming at Latnjajaure Field Station, Northern Sweden.</em> <a href="https://doi.org/10.1139/as-2020-0042" class="external-link uri">https://doi.org/10.1139/as-2020-0042</a>. Research-Article. Apr. 2021. DOI: 10.1139/AS-2020-0042. (Visited on 04/05/2021).</p>
<p>[86] L. Seep, Z. Razaghi-Moghadam, and Z. Nikoloski. “Reaction Lumping in Metabolic Networks for Application with Thermodynamic Metabolic Flux Analysis”. In: <em>Scientific Reports</em> 11.1 (Apr. 2021), p. 8544. ISSN: 2045-2322. DOI: 10.1038/s41598-021-87643-8. (Visited on 04/23/2021).</p>
<p>[87] O. Seppälä. “Spatial and Temporal Drivers of Soil Respiration in a Tundra Environment”. MA Thesis. FACULTY OF SCIENCE DEPARTMENT OF GEOSCIENCES AND GEOGRAPHY GEOGRAPHY: UNIVERSITY OF HELSINKI, 2020.</p>
<p>[88] L. Shah, C. A. Arnillas, and G. B. Arhonditsis. “Characterizing Temporal Trends of Meteorological Extremes in Southern and Central Ontario, Canada”. In: <em>Weather and Climate Extremes</em> (Jan. 2022), p. 100411. ISSN: 2212-0947. DOI: 10.1016/j.wace.2022.100411. (Visited on 01/29/2022).</p>
<p>[89] S. A. Simon, K. Schmidt, L. Griesdorn, et al. <em>Dancing the Nanopore Limbo - Nanopore Metagenomics from Small DNA Quantities for Bacterial Genome Reconstruction</em>. Feb. 2023. DOI: 10.1101/2023.02.16.527874. (Visited on 02/20/2023).</p>
<p>[90] E. Smertina, L. M. Keller, N. Huang, et al. “First Detection of Benign Rabbit Caliciviruses in Chile”. In: <em>Viruses</em> 16.3 (Mar. 2024), p. 439. ISSN: 1999-4915. DOI: 10.3390/v16030439. (Visited on 03/20/2024).</p>
<p>[91] C. C. Smith, S. Entwistle, C. Willis, et al. “Landscape and Selection of Vaccine Epitopes in SARS-CoV-2”. In: <em>bioRxiv</em> (Jun. 2020). DOI: 10.1101/2020.06.04.135004. (Visited on 03/03/2021).</p>
<p>[92] J. H. Sogin and R. W. Worobo. “Primary Metabolites and Microbial Diversity in Commercial Kombucha Products”. In: <em>Fermentation</em> 10.8 (Aug. 2024), p. 385. ISSN: 2311-5637. DOI: 10.3390/fermentation10080385. (Visited on 08/04/2024).</p>
<p>[93] F. St-Onge, M. Javanray, A. Pichet Binette, et al. “Functional Connectome Fingerprinting across the Lifespan”. In: <em>Network Neuroscience</em> (May. 2023), pp. 1-55. ISSN: 2472-1751. DOI: 10.1162/netn_a_00320. (Visited on 05/09/2023).</p>
<p>[94] T. L. Stach, J. Starke, F. Bouderka, et al. <em>A Uniform Stress Response of Stream Microbiomes in the Hyporheic Zone across North America</em>. Feb. 2025. DOI: 10.1101/2025.02.16.638492. (Visited on 02/24/2025).</p>
<p>[95] L. Stoler-Barak, D. Schmiedel, A. Sarusi-Portuguez, et al. “SMARCA5-mediated Chromatin Remodeling Is Required for Germinal Center Formation”. In: <em>Journal of Experimental Medicine</em> 221.11 (Sep. 2024), p. e20240433. ISSN: 0022-1007. DOI: 10.1084/jem.20240433. (Visited on 09/25/2024).</p>
<p>[96] S. N. Thiede, E. S. Snitkin, W. Trick, et al. “Genomic Epidemiology Suggests Community Origins of Healthcare-Associated USA300 MRSA”. In: <em>The Journal of Infectious Diseases</em> (Feb. 2022), p. jiac056. ISSN: 0022-1899. DOI: 10.1093/infdis/jiac056. (Visited on 02/26/2022).</p>
<p>[97] A. V. Thorn, F. M. Aarestrup, and P. Munk. “Flankophile: A Bioinformatic Pipeline for Prokaryotic Genomic Synteny Analysis”. In: <em>Microbiology Spectrum</em> (Dec. 2023), p. e0241323. ISSN: 2165-0497. DOI: 10.1128/spectrum.02413-23.</p>
<p>[98] A. Torres-Espín, A. Chou, J. R. Huie, et al. “Reproducible Analysis of Disease Space via Principal Components Using the Novel R Package syndRomics”. In: <em>eLife</em> 10 (Jan. 2021). Ed. by M. Zaidi and M. Barton, p. e61812. ISSN: 2050-084X. DOI: 10.7554/eLife.61812. (Visited on 03/03/2021).</p>
<p>[99] Y. Uneno, K. Fukuyama, A. Nishimura, et al. “Barriers and Facilitators to the Implementation of an Electronic Patient-Reported Outcome System at Cancer Hospitals in Japan”. In: <em>Cureus</em> 16.4 (Apr. 2024). ISSN: 2168-8184. DOI: 10.7759/cureus.58611. (Visited on 04/23/2024).</p>
<p>[100] S. Valenzuela. “Social Deprivation and Multimorbidity Among Community-Based Health Center Patients in the United States”. In: <em>Preventing Chronic Disease</em> 21 (2024). ISSN: 1545-1151. DOI: 10.5888/pcd21.240060. (Visited on 10/01/2024).</p>
<p>[101] I. Veseli, M. A. DeMers, Z. S. Cooper, et al. “Digital Microbe: A Genome-Informed Data Integration Framework for Team Science on Emerging Model Organisms”. In: <em>Scientific Data</em> 11.1 (Sep. 2024), p. 967. ISSN: 2052-4463. DOI: 10.1038/s41597-024-03778-z. (Visited on 09/10/2024).</p>
<p>[102] C. Wang, X. Zhao, H. Zhang, et al. “Comprehensive Analysis of Immune-Related Genes Associated with the Microenvironment of Patients with Unexplained Infertility”. In: <em>Annals of Translational Medicine</em> 11.2 (Jan. 2023), pp. 84-84. ISSN: 2305-5847, 2305-5839. DOI: 10.21037/atm-22-5810. (Visited on 02/12/2023).</p>
<p>[103] L. Weidenauer and M. Quadroni. “Phosphorylation in the Charged Linker Modulates Interactions and Secretion of Hsp90<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>β</mi><annotation encoding="application/x-tex">\beta</annotation></semantics></math>”. In: <em>Cells</em> 10.7 (Jul. 2021), p. 1701. DOI: 10.3390/cells10071701. (Visited on 07/08/2021).</p>
<p>[104] D. Wendisch, O. Dietrich, T. Mari, et al. “SARS-CoV-2 Infection Triggers Profibrotic Macrophage Responses and Lung Fibrosis”. In: <em>Cell</em> (Nov. 2021). ISSN: 0092-8674. DOI: 10.1016/j.cell.2021.11.033. (Visited on 12/11/2021).</p>
<p>[105] D. Western, J. Timsina, L. Wang, et al. “Proteogenomic Analysis of Human Cerebrospinal Fluid Identifies Neurologically Relevant Regulation and Implicates Causal Proteins for Alzheimer’s Disease”. In: <em>Nature Genetics</em> (Nov. 2024), pp. 1-13. ISSN: 1546-1718. DOI: 10.1038/s41588-024-01972-8. (Visited on 11/15/2024).</p>
<p>[106] R. Woyda, A. Oladeinde, and Z. Abdo. “Chicken Production and Human Clinical Escherichia Coli Isolates Differ in Their Carriage of Antimicrobial Resistance and Virulence Factors”. In: <em>Applied and Environmental Microbiology</em> 0.0 (Jan. 2023), pp. e01167-22. DOI: 10.1128/aem.01167-22. (Visited on 01/25/2023).</p>
<p>[107] R. J. Wright, M. G. I. Langille, and T. R. Walker. “Food or Just a Free Ride? A Meta-Analysis Reveals the Global Diversity of the Plastisphere”. In: <em>The ISME Journal</em> 15.3 (Mar. 2021), pp. 789-806. ISSN: 1751-7370. DOI: 10.1038/s41396-020-00814-9. (Visited on 03/03/2021).</p>
<p>[108] P. Wu, C. Chang, G. Zhu, et al. “Network Pharmacology Study of Bioactive Components and Molecular Mechanisms of the Glycoside Fraction from <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&lt;</mo><annotation encoding="application/x-tex">&lt;</annotation></semantics></math>em<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&gt;</mo><annotation encoding="application/x-tex">&gt;</annotation></semantics></math>Picrorhiza Scrophulariiflora<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&lt;</mo><annotation encoding="application/x-tex">&lt;</annotation></semantics></math>/Em<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>&gt;</mo><annotation encoding="application/x-tex">&gt;</annotation></semantics></math> Against Experimental Colitis”. In: <em>Drug Design, Development and Therapy</em> 17 (May. 2023), pp. 1531-1546. DOI: 10.2147/DDDT.S407339. (Visited on 05/28/2023).</p>
<p>[109] T. Wyenberg-Henzler, R. T. Patterson, and J. C. Mallon. “Ontogenetic Dietary Shifts in North American Hadrosaurids”. In: <em>Cretaceous Research</em> (Feb. 2022), p. 105177. ISSN: 0195-6671. DOI: 10.1016/j.cretres.2022.105177. (Visited on 02/26/2022).</p>
<p>[110] L. Xie, H. Liu, Z. You, et al. “Comprehensive Spatiotemporal Mapping of Single-Cell Lineages in Developing Mouse Brain by CRISPR-based Barcoding”. In: <em>Nature Methods</em> (Jul. 2023), pp. 1-12. ISSN: 1548-7105. DOI: 10.1038/s41592-023-01947-3. (Visited on 07/24/2023).</p>
<p>[111] M. Xie, B. Cheng, S. Yu, et al. “Cuproptosis-Related MiR-21-5p/FDX1 Axis in Clear Cell Renal Cell Carcinoma and Its Potential Impact on Tumor Microenvironment”. In: <em>Cells</em> 12.1 (Dec. 2022), p. 173. ISSN: 2073-4409. DOI: 10.3390/cells12010173.</p>
<p>[112] Y. Xu, M. Price, P. Que, et al. “Ecological Predictors of Interspecific Variation in Bird Bill and Leg Lengths on a Global Scale”. In: <em>Proceedings of the Royal Society B: Biological Sciences</em> 290.2003 (Jul. 2023), p. 20231387. DOI: 10.1098/rspb.2023.1387. (Visited on 07/30/2023).</p>
<p>[113] A. Yan, J. Butcher, D. Mack, et al. “Virome Sequencing of the Human Intestinal Mucosal-Luminal Interface”. In: <em>Frontiers in Cellular and Infection Microbiology</em> 10 (Oct. 2020). ISSN: 2235-2988. DOI: 10.3389/fcimb.2020.582187. (Visited on 03/03/2021).</p>
<p>[114] P. Zannini, F. Frascaroli, J. Nascimbene, et al. “Sacred Natural Sites and Biodiversity Conservation: A Systematic Review”. In: <em>Biodiversity and Conservation</em> (Sep. 2021). ISSN: 1572-9710. DOI: 10.1007/s10531-021-02296-3. (Visited on 10/04/2021).</p>
<p>[115] D. Zhang, W. Wang, C. Song, et al. “Comparative Genomic Study of Non-typeable Haemophilus Influenzae in Children with Pneumonia and Healthy Controls”. In: <em>iScience</em> 0.0 (Nov. 2024). ISSN: 2589-0042. DOI: 10.1016/j.isci.2024.111330. (Visited on 11/08/2024).</p>
<p>[116] H. L. Zhang, K. J. Gontjes, J. H. Han, et al. “Characterization of Resistance to Newer Antimicrobials among Carbapenem-Resistant Klebsiella Pneumoniae in the Post-Acute-Care Setting”. In: <em>Infection Control &amp; Hospital Epidemiology</em> (Jul. 2022), pp. 1-4. ISSN: 0899-823X, 1559-6834. DOI: 10.1017/ice.2022.185. (Visited on 08/01/2022).</p>
<p>[117] X. Zhang, X. Yu, Z. Yu, et al. “Network Pharmacology and Bioinformatics to Identify Molecular Mechanisms and Therapeutic Targets of Ruyi Jinhuang Powder in the Treatment of Monkeypox”. In: <em>Medicine</em> 102.17 (Apr. 2023), p. e33576. DOI: 10.1097/MD.0000000000033576. (Visited on 05/02/2023).</p>
[118] P. Zhu, W. Liu, X. Zhang, et al. “Correlated Evolution of Social Organization and Lifespan in Mammals”. In: <em>Nature Communications</em> 14.1 (Jan. 2023), p. 372. ISSN: 2041-1723. DOI: 10.1038/s41467-023-35869-7. (Visited on 02/04/2023).
</details>
</div>
<div class="section level2">
<h2 id="usage">Usage<a class="anchor" aria-label="anchor" href="#usage"></a>
</h2>
<p>The main function is <code><a href="reference/new_scale.html">new_scale()</a></code> and its aliases <code><a href="reference/new_scale.html">new_scale_color()</a></code> and <code><a href="reference/new_scale.html">new_scale_fill()</a></code>. When added to a plot, every geom added after them will use a different scale.</p>
<p>As an example, let’s overlay some measurements over a contour map of topography using the beloved <code>volcano</code>.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://eliocamp.github.io/ggnewscale/">ggnewscale</a></span><span class="op">)</span></span>
<span><span class="co"># Equivalent to melt(volcano)</span></span>
<span><span class="va">topography</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/expand.grid.html" class="external-link">expand.grid</a></span><span class="op">(</span>x <span class="op">=</span> <span class="fl">1</span><span class="op">:</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">volcano</span><span class="op">)</span>,</span>
<span>                          y <span class="op">=</span> <span class="fl">1</span><span class="op">:</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">ncol</a></span><span class="op">(</span><span class="va">volcano</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">topography</span><span class="op">$</span><span class="va">z</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="va">volcano</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># point measurements of something at a few locations</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/Random.html" class="external-link">set.seed</a></span><span class="op">(</span><span class="fl">42</span><span class="op">)</span></span>
<span><span class="va">measurements</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span>x <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/Uniform.html" class="external-link">runif</a></span><span class="op">(</span><span class="fl">30</span>, <span class="fl">1</span>, <span class="fl">80</span><span class="op">)</span>,</span>
<span>                           y <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/Uniform.html" class="external-link">runif</a></span><span class="op">(</span><span class="fl">30</span>, <span class="fl">1</span>, <span class="fl">60</span><span class="op">)</span>,</span>
<span>                           thing <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/Normal.html" class="external-link">rnorm</a></span><span class="op">(</span><span class="fl">30</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span>mapping <span class="op">=</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span><span class="va">x</span>, <span class="va">y</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_contour.html" class="external-link">geom_contour</a></span><span class="op">(</span>data <span class="op">=</span> <span class="va">topography</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>z <span class="op">=</span> <span class="va">z</span>, color <span class="op">=</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes_eval.html" class="external-link">stat</a></span><span class="op">(</span><span class="va">level</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="co"># Color scale for topography</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_viridis.html" class="external-link">scale_color_viridis_c</a></span><span class="op">(</span>option <span class="op">=</span> <span class="st">"D"</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="co"># geoms below will use another color scale</span></span>
<span>  <span class="fu"><a href="reference/new_scale.html">new_scale_color</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span>data <span class="op">=</span> <span class="va">measurements</span>, size <span class="op">=</span> <span class="fl">3</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>color <span class="op">=</span> <span class="va">thing</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="co"># Color scale applied to geoms added after new_scale_color()</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_viridis.html" class="external-link">scale_color_viridis_c</a></span><span class="op">(</span>option <span class="op">=</span> <span class="st">"A"</span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning: `stat(level)` was deprecated in ggplot2 3.4.0.</span></span>
<span><span class="co">#&gt; ℹ Please use `after_stat(level)` instead.</span></span>
<span><span class="co">#&gt; This warning is displayed once every 8 hours.</span></span>
<span><span class="co">#&gt; Call `lifecycle::last_lifecycle_warnings()` to see where this warning was</span></span>
<span><span class="co">#&gt; generated.</span></span></code></pre></div>
<p><img src="reference/figures/README-unnamed-chunk-3-1.png"><!-- --></p>
<p>If you want to create new scales for other <code>aes</code>, you can call <code>new_scale</code> with the name of the <code>aes</code>. For example, use</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="reference/new_scale.html">new_scale</a></span><span class="op">(</span><span class="st">"linetype"</span><span class="op">)</span></span></code></pre></div>
<p>to add multiple linetype scales.</p>
</div>
</div>
  </main><aside class="col-md-3"><div class="links">
<h2 data-toc-skip>Links</h2>
<ul class="list-unstyled">
<li><a href="https://cloud.r-project.org/package=ggnewscale" class="external-link">View on CRAN</a></li>
<li><a href="https://github.com/eliocamp/ggnewscale/" class="external-link">Browse source code</a></li>
<li><a href="https://github.com/eliocamp/ggnewscale/issues" class="external-link">Report a bug</a></li>
</ul>
</div>

<div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="LICENSE.html">Full license</a></li>
<li><small><a href="https://www.r-project.org/Licenses/GPL-3" class="external-link">GPL-3</a></small></li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing ggnewscale</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Elio Campitelli <br><small class="roles"> Maintainer, author </small> <a href="https://orcid.org/0000-0002-7742-9230" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a>  </li>
</ul>
</div>

<div class="dev-status">
<h2 data-toc-skip>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/eliocamp/ggnewscale/actions" class="external-link"><img src="https://github.com/eliocamp/ggnewscale/workflows/R-CMD-check/badge.svg" alt="R build status"></a></li>
<li><a href="https://app.codecov.io/gh/eliocamp/ggnewscale?branch=master" class="external-link"><img src="https://codecov.io/gh/eliocamp/ggnewscale/branch/master/graph/badge.svg" alt="Codecov test coverage"></a></li>
<li><a href="https://zenodo.org/badge/latestdoi/161934647" class="external-link"><img src="https://zenodo.org/badge/161934647.svg" alt="DOI"></a></li>
<li><a href="https://cran.r-project.org/package=ggnewscale" class="external-link"><img src="http://www.r-pkg.org/badges/version/ggnewscale" alt="CRAN status"></a></li>
<li><a href="https://github.com/eliocamp/ggnewscale/actions/workflows/R-CMD-check.yaml" class="external-link"><img src="https://github.com/eliocamp/ggnewscale/actions/workflows/R-CMD-check.yaml/badge.svg" alt="R-CMD-check"></a></li>
</ul>
</div>

  </aside>
</div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Elio Campitelli.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
